composer.json

```diff
@@ -23,7 +23,8 @@
         "drush/drush": "~8.0",
         "drupal/console": "~1.0",
         "drupal/devel": "~1.0",
-        "drupal/businessgroup_zymphonies_theme": "8.1.0"
+        "drupal/businessgroup_zymphonies_theme": "~1.0",
+        "drupal/google_map_field": "^1.2"
```

Comment:
Here you should specify a fix version of the module, including the core number. Ex: 8.1.2